src/store/actions/api.actions.js

```diff
@@ -248,13 +239,14 @@ export const fetchEndpoint = pathname => async dispatch => {
 
             return plugin;
         });
-
         const lens = R.lensPath(['plugins']);
         const preparedApi = R.set(lens, preparedPlugins, response.data);
 
         dispatch(getEndpointSuccess(preparedApi, response.data));
```

Comment:
I just wonder if in these cases we could compose 🤔 
```js
R.compose(
  dispatch,
  getEndpointSuccess
)(preparedApi, response.data)
```

Reply:
@rafaeljesus , I'll check this out! 👍